_posts/2024-07-22-pyopensci-at-scipy-2024.md

```diff
@@ -0,0 +1,463 @@
+pyOpenSci's mission of supporting open science forward.
+
+<figure>
+  <img src="{{ site.baseurl }}/images/blog/2024/july/scipy-24-pyopensci-package-submission-growth.png" alt="A bar plot with purple bars titled - number of submissions by quarter per year. The plot shows the number of submissions that pyOS has received since it started peer review in 2019. We started with only a few submissions per quarter but the numbers have grown to about 13 per quarter on average in the 2024. " style="width: 100%;" />
+  <figcaption><i class="fa-solid fa-heart-pulse"></i> pyOpenSci runs an open software peer review process. Above you can see how submissions have increased since we started in 2019. I started full time on pyOpenSci in late 2022. <i class="fa-solid fa-heart-pulse"></i>.</figcaption>
+</figure>
+
+In the end, even though I wasn't quite as prepared as I wanted to be, my talk
+and the workshop went great! This was a lesson for me in understanding that
+things can be less than perfect and still go well.
+
+More on all of that below. But first--what is the SciPy Conference?
+
+## About the SciPy Conference
+
+If you haven't attended this meeting before, let me give you the rundown. The
+SciPy Conference is an annual event dedicated to celebrating and learning more
+about scientific computing using the Python programming language. The meeting
+started back in [~2002](https://web.archive.org/web/20021030003450/http://www.scipy.org/site_content/scipy02/). The early meetings were run by EnThought and
+were smaller in size and scope. Over the years, SciPy has evolved significantly,
+mirroring the [🚀 explosive growth of the Python ecosystem 🚀](https://stackoverflow.blog/2023/01/26/comparing-tag-trends-with-our-most-loved-programming-languages/)
+in scientific computing and data science.
+
+Today, the SciPy Conference, hosted by [NumFocus](https://numfocus.org/),
+attracts over 700 attendees from diverse scientific fields such as astronomy,
+biology, geophysics, and more. SciPy is much more than just a workshop.
+
+SciPy features:
+
+* Incredible keynote speakers, such as the inspirational
+  [Carol Willing](https://www.youtube.com/watch?v=s-W-UvGgDco).
+* Talks from the community about tools and approaches.
+* Poster sessions that allow presenters to directly engage and discuss cool
+  work with the meeting attendees.
+* Tutorials led by community members (and pyOpenSci) that cover important
+  scientific data processing and analysis-related Python skills.
+* Sprints: collaborative coding sessions where people come to contribute to
+  and learn about open-source projects.
+
+I'd be remiss if I didn't mention the lightning talk sessions. During lightning
+talks, people throw their names into a literal bucket. If their name is selected,
+they are given 5 minutes (or less if the dice are rolled) to talk about a topic
+they love. At SciPy, there are often off-beat distractions from the moderators
```

Comment:
```suggestion
they love. At SciPy, there are often off-beat distractions from the moderators
```
is there a better description than distraction? this current sentence feels like people are giving lightning talks, and then the moderators interrupt//distract the audience with something completely off-topic. if that's what's happening, then it would be great to get some gentler language to help the reader prepare for this, and to make it seem fun and accessible.